### ISSUE TITLE
[Screen Reader - Open and close New Tab]: Same name as "Windows Powershell" is defined for all the tabs.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8473]
**User Experience:**
Users who rely on screen reader would be impacted and not able to get which tab is open as same name "Windows Powershell" is defined for all the tabs.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0
Tool: Accessibility Insights for Windows

**Repro Steps:**
1. Open Windows terminal app.
2. Press "Ctrl+Shift+T"  to open multiple tabs.
3. Now run accessibility insight tool on the application.
4. Observe the issue.

**Actual Result:**
Same name as "Windows Powershell" is defined for all the tabs. So when screen reader focus navigates to any tab then narrator just announces it as "**Windows powershell editing space**".

**Expected Result:**
Unique name should be defined for each tabs. 
For e.g. **Name should be defined as "Windows powershell 1" for 1st tab item and so on**.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B248054a6-5e68-4771-9e1e-242fb5025730%7D&cid=e71aec14-4173-423e-aecb-2472bbc751e8

![MAS4 1 2_AI_Tabs_same name defined_Details](https://user-images.githubusercontent.com/53220700/100990886-67dfcd00-3578-11eb-9d8b-720991bb98e5.png)
![MAS4 1 2_AI_Tabs_same name defined_How to fix](https://user-images.githubusercontent.com/53220700/100990894-6910fa00-3578-11eb-955c-4e972ad4f06c.png)
[MAS4.1.2_Same name defind for each tab.zip](https://github.com/microsoft/terminal/files/5635053/MAS4.1.2_Same.name.defind.for.each.tab.zip)

**Comment by zadjii-msft** [comment]
Isn't this by-design? The client application (`powershell.exe`) is responsible for setting the title of the tab, not the Terminal. What does Chromium do if the user has a bunch of github.com tabs open?

**Comment by DHowett** [comment]
@Chandresh1007 yes, please clarify. What happens in a normal web browser when the same tab is opened multiple times?

**Comment by Chandresh1007** [comment]
@DHowett We agree that web browsers also don't handled multiple tabs with same name in a different way, but here, it's easy to fix the bug by prefixing the string such as: "tab 1", "tab2" in front of the console processes such as 1st powershell session etc.
In current scenario, user won't be able to differentiate between different tabs while switching.
Also, a command should be added so that user can rename the tab as per his/her comfort.

**Comment by zadjii-msft** [comment]
> Also, a command should be added so that user can rename the tab as per his/her comfort.

That already exists in a bunch of different forms. There's: 
* the [`renameTab` action](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#rename-tab)
* the [`openTabRenamer` action](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/actions#open-tab-rename-text-box-preview)
* The user can right-click the tab to open the renamer: ![image](https://user-images.githubusercontent.com/18356694/101354319-900e4b00-385a-11eb-8218-ee1c1361c7fb.png)
* They can just double-click the tab ![image](https://user-images.githubusercontent.com/18356694/101354337-98ff1c80-385a-11eb-96cf-093138a538a7.png)
* [They could also set the title from their shell](https://docs.microsoft.com/en-us/windows/terminal/tutorials/tab-title)

**Comment by Chandresh1007** [comment]
@zadjii-msft But how do keyboard user can access that as Keyboard tab focus does not navigate to the Tabs. Is there any key defined for renaming the tab?

**Comment by zadjii-msft** [comment]
Not by default, but the user can bind any key they want to `renameTab`/`openTabRenamer`. They can also use the action as it's bound in the Command Palette to access the action even without binding a key to it:

 
![open-tab-renamer-from-cmdpal](https://user-images.githubusercontent.com/18356694/101372941-09fdfe80-3872-11eb-97cc-982464e76257.gif)

**Comment by v-akkak** [comment]
@zadjii-msft 
It's absolutely fine, however, we'll be very thankful to you folks if a default keybinding could be added to this action.
It will enhance the user experience as user won't have to manually asign the keybinding since it is quite a common action.

> Not by default, but the user can bind any key they want to `renameTab`/`openTabRenamer`. They can also use the action as it's bound in the Command Palette to access the action even without binding a key to it:
> 
> ![open-tab-renamer-from-cmdpal](https://user-images.githubusercontent.com/18356694/101372941-09fdfe80-3872-11eb-97cc-982464e76257.gif)

**Comment by Chandresh1007** [comment]
Feature team does not consider this a bug. It is the same as navigating between multiple PowerShell windows.

It may be a duplicate of #8474 (“screen reader does not announce position of tabs.”)

Hence Closing the bug.
